The RESOLVE_SCM_NUMBER ag-text string needs to be adjusted so it
prints out correctly on 32-bit SPARC machines. It was just plain
luck that this worked correctly on 32-bit x86 (little-endian)
machines, as the four least significant bytes of the returned value
contained the correct value.

Rather than adjust the majority of the 404 string offsets in this
file, we simply loose one of the spaces in the following string.

This patch has been sent upstream to the autogen author/maintainer.

--- autogen-5.16.2/agen5/ag-text.c.orig	2013-05-28 18:52:53.386918600 -0700
+++ autogen-5.16.2/agen5/ag-text.c	2013-05-28 19:09:56.084583327 -0700
@@ -65,8 +65,8 @@
             "%s\0"
 /*   646 */ "** BOGUS **\0"
 /*   658 */ "break\0"
-/*   664 */ "%ld\0"
-/*   668 */ "fserr %d: cannot %s %s:  %s\n\0"
+/*   664 */ "%lld\0"
+/*   669 */ "fserr %d: cannot %s %s: %s\n\0"
 /*   697 */ "duplicate make target\0"
 /*   719 */ "unknown dependency type:  %s\0"
 /*   748 */ "Content-type: text/plain\n\n"
--- autogen-5.16.2/agen5/ag-text.h.orig	2013-05-28 18:53:06.411186250 -0700
+++ autogen-5.16.2/agen5/ag-text.h	2013-05-28 19:09:34.497154304 -0700
@@ -86,8 +86,8 @@
 #define BREAK_STR                    (ag_text_strtable+658)
 #define BREAK_STR_LEN                5
 #define BUILD_DEFS_LONG_FMT          (ag_text_strtable+664)
-#define BUILD_DEFS_LONG_FMT_LEN      3
-#define CANNOT_FMT                   (ag_text_strtable+668)
+#define BUILD_DEFS_LONG_FMT_LEN      4
+#define CANNOT_FMT                   (ag_text_strtable+669)
 #define CANNOT_FMT_LEN               28
 #define CFGDEP_DUP_TARGET_MSG        (ag_text_strtable+697)
 #define CFGDEP_DUP_TARGET_MSG_LEN    21
